src/components/components/EditNewsForm.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import apiPostagem from "../../services/apiPostagem";
+import { withRouter } from "react-router-dom";
+import "./editNewsForm.css";
+import apiNoticias from "../../services/apiNoticias";
+
+
+
+class EditNewsForm extends React.Component {
+
+    constructor(props){
+      super(props);
+      this.state = {
+        title: "",
+        subtitle: "",
+        text:  "",
+        image1: "",
+        image2: "",
+        image3: "",
+        post_id: "",
```

Comment:
Codacy found an issue: [Identifier 'post_id' is not in camel case.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6404581)